src/components/info-box.pcss

```diff
@@ -0,0 +1,60 @@
+.info-box {
+  align-items: center;
+  background-color: white;
+  border: 1px solid #ccc;
+  border-radius: 0.5rem;
+  box-shadow: 2px 2px 10px #ccc;
+  color: #404040;
+  display: flex;
+  font-size: 2em;
+  height: 7em;
+  justify-content: center;
+  left: 0;
+  margin-bottom: 2em;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 30em;
+  position: absolute;
+  right: 0;
+  width: fit-content;
+  z-index: 50;
+
+  p {
+    margin-right: 1.5em;
+  }
+}
+
+.visible {
+  animation: box-in 0.7s;
+  bottom: 0;
+  transition: bottom 0.6s ease-in-out;
+}
+
+.hidden {
+  bottom: -15%;
+  transition: bottom 0.8s ease-in-out;
```

Comment:
It was a tiiny bit too slow for my liking, I think I read somewhere that around 300-500ms is max before people start noticing the animation instead of the content :)

Reply:
Decreasing to 0.5!